docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+# Default Reference Types
+
+We've mentioned throughout this tutorial what the default reference capabilities are for types and classes. However, there's a good chance those defaults passed over your head while you were focused on understanding capabilities themselves.
+
+First, there is nothing saying you have to use default capabilities. Especially as you start writing your first Pony programs, you'll probably find it very educational to explicitly specify the capabilities for all your classes, actors, functions, parameters and fields. However, it's not _writing_ code that makes default capabilities confusing; it's _reading_ that code. Hopefully keeping this summary open will help you navigate pony source code until you've become used to the defaults!
+
+## Reference type summary
+
+We go into detail of the default types below, but here's a table summarizing them:
+
+| Item                    | Default Capability | Explanation                                                             |
```

Comment:
I find numerous things in this table confusing. I understand it is trying to be expansive but I think it ends up being rather confusing in some places, like listing defaults for things and then saying "well but only this other type can have different capabilities".

I think a distinction needs to be made between "default" as in "you can override" and "this is what it is" and only discuss the former and not mix in the latter in the same section.

Reply:
That’s been tripping me up all along. Perhaps two separate tables.

Reply:
I split it into two tables, but I still don't love it. I don't have any specific ideas, so I'll leave this as an open question.